index.html

```diff
@@ -8428,7 +8428,7 @@ <h3>Changes since the
     This brings the PNG specification into alignment
     with widely deployed industry practice.
   </li>
-  <li>Added the <a class="chunk" href="#11cICP">cICP</a> chunk,
+  <li>Added the <a class="chunk" href="#cICP-chunk">cICP</a> chunk,
```

Comment:
This is unrelated to the change. But it is small enough that I didn't really want to create a whole pull request just to fix it.